### ISSUE TITLE
All caps accessibility

### ISSUE THREAD

**Issue by ehoops** [chanzuckerberg/sci-components#36]
Avoiding all caps font improves readability/accessibility: https://accessibility.huit.harvard.edu/design-readability
More information about this is here: https://github.com/humanmade/hm-pattern-library/issues/75#issuecomment-235672885

In order to create a "pit of success" where the easiest thing to do is accessible, we should consider removing the all-caps font mixins from the design system.

**Comment by happyimadesignr** [comment]
Hey @ehoops are you referring to this mixin?
![Screen Shot 2021-07-08 at 12 46 25 PM](https://user-images.githubusercontent.com/8314555/124981515-831d2a00-dfea-11eb-89c2-1bc53cc831f2.png)

For IDseq we mostly only use it for text buttons and some subheaders, which I think is ok. I agree we should not use it for any sentences or long phrases. Looks like it would help to include some usage rules around this mixin.

**Comment by ehoops** [comment]
Yes, that's the one.  I think we are pretty good in general about not using it for long phrases.  Are there alternatives we could use instead for the buttons and subheaders?  I know Meta switched some of them (although since I left some crept back in 😭 ) to be bold instead: 
<img width="397" alt="Screen Shot 2021-07-08 at 12 52 30 PM" src="https://user-images.githubusercontent.com/8015489/124982378-7a792380-dfeb-11eb-8a82-0a6b7cb37c6e.png">

**Comment by ehoops** [comment]
Another thing to note is that products can technically do whatever they want, but the design system should be a place to get best practices.

**Comment by happyimadesignr** [comment]
We could probably figure out an alternative for the headers and the minimal dropdown input. It's challenging on IDseq b/c it is so dense with information already, the caps help with distinguishing it and with hierarchy. We did switch some of our tags from caps to title case at one point b/c it was getting too hard to read.

cc @Janeece since we'll be working on the minimal dropdown input. I think it would make sense to make the SDS minimal dropdown not have caps and then style the IDseq specific one with caps. Then the SDS would be closer to @hthomas-czi 's design more.

I don't think we could switch our minimal buttons over because the caps distinguish them from text links.

![Screen Shot 2021-07-08 at 5 25 09 PM](https://user-images.githubusercontent.com/8314555/125005775-752ecf80-e011-11eb-8615-5f69f62dbb8d.png)

**Comment by ehoops** [comment]
It seems like a good compromise to have the SDS not use caps and leave it up to the teams to override when necessary.

**Comment by tihuan** [comment]
I remember @happyimadesignr and I were talking about Infectious Disease Team having their own library on top of SDS, so that could be a good place for the override?

**Comment by mmmmmmaya** [comment]
It sounds like we've reached consensus here.
